### ISSUE TITLE
Opening and dismissing Revision Selector with screen reader/keyboard does not return focus to previously focused element

### ISSUE THREAD

**Issue by dcalhoun** [Automattic/simplenote-electron#2802]
The typical approach for managing focus with dialogs is that focus is returned to the previously focused element once the dialog is closed/dismissed. I.e. closing a modal should return focus back to the button used to open the modal. This allows the screen reader/keyboard user to continue from the last focus set.

Currently, the focus is unexpectedly placed on the editor when the Revision Selector is dismissed. This is related to the work in #2531 as mentioned in https://github.com/Automattic/simplenote-electron/pull/2531#pullrequestreview-607705670.

### Expected
The History button regains focus.

### Observed
The note editor gains focus.

### Reproduced
1. Launch Simplenote.
2. Enable screen reader and move focus to History button -OR- click outside of the editor (to remove focus from editor) and press <kbd>Tab</kbd> until the History button is focused.
3. Press <kbd>Return</kbd> to click the History button.
4. Press <kbd>Esc</kbd> to dismiss the Revision Selector.

https://user-images.githubusercontent.com/438664/112163272-df367c80-8bba-11eb-8aeb-6c13b1fbb656.mov

### Where did you see the bug

- System Make: Apple
- System Model: MacBook Pro
- OS: macOS
- OS version: 11.2.3 
- Browser (if applicable): Safari
- Browser version (if applicable): 14.0.3
- Simplenote app version: 2.9.0-beta1
